src/js/components/DeploymentComponent.jsx

```diff
@@ -97,16 +96,16 @@ var DeploymentComponent = React.createClass({
         </td>
         <td>
           <ul className="list-unstyled">
-            {lazy(model.currentActions).map(function (action) {
+            {model.currentActions.map(function (action) {
               return <li key={action.app}>{action.app}</li>;
-            }).value()}
+            })}
```

Comment:
I tend to shy away having logic inside templates.
Can we move this to like #89 or so? We can then assign the return value to a variable and then use that here.
For example:
```
var currentActions = model.currentActions.map(function (action) {
  return <li key={action.app}>{action.app}</li>;
});
``
Then we can use `currentActions` in the template.